rhaegal/FromWork/GenericJoin.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/python
+##########################################################
+# join all part files in a dir created by split.py.  
+# This is roughly like a 'cat fromdir/* > tofile' command
+# on unix, but is a bit more portable and configurable,
+# and exports the join operation as a reusable function.
+# Relies on sort order of file names: must be same length.
+# Could extend split/join to popup Tkinter file selectors.
+##########################################################
+
+import os, sys
+readsize = 1024
+
+def join(fromdir, tofile):
+    output = open(tofile, 'wb')
+    parts  = os.listdir(fromdir)
+    parts.sort(  )
+    for filename in parts:
+        if not "MD5" in filename:
+            if "part" in filename:
+                filepath = os.path.join(fromdir, filename)
+                fileobj  = open(filepath, 'rb')
```

Comment:
try to use a context manager here so you don't have to worry about closing you file object:
`with open(filepath, 'rb') as fileobj`

https://docs.python.org/2/reference/compound_stmts.html#the-with-statement